src/collectives/device/msccl_interpreter.h

```diff
@@ -143,13 +142,14 @@ namespace {
             if (thisNelem < nthreads){
               if (tid < thisNelem){
                 dstoffset = gridOffset + (ssize_t) (msccltran->dstoffset+c) * sizePerMscclChunk;
-                T o = load(dstPointer + dstoffset+tid);
+                T* region = dstPointer + dstoffset +tid;
```

Comment:
instead of region, can you please call it dst_index

Reply:
sure I can.